contentcuration/contentcuration/fixtures/admin_user.json

```diff
@@ -0,0 +1,23 @@
+[
+{
+  "model": "contentcuration.user",
+  "pk": 1,
+  "fields": {
+    "password": "pbkdf2_sha256$24000$mst3QQzzUZdc$W2p+jovz/vp1I1oP4tP8x3H4Cp05Rrxm9ZVcrYFphKM=",
+    "last_login": "2018-02-18T19:19:59.197Z",
+    "is_superuser": true,
+    "email": "content@learningequality.org",
```

Comment:
Will this actually send an email? Don't want to spam the content@ email with a lot of test emails